types/office-js-preview/index.d.ts

```diff
@@ -13958,6 +13963,13 @@ declare namespace Office {
          *
          * The add-in should use the ewsUrl property to determine the correct URL to use when making EWS calls.
          *
+         * You can pass the token and an attachment identifier or item identifier to a third-party system. The third-party system uses the token as a
```

Comment:
The grouping of the "and" and "or" isn't 100% clear. Consider: "You can pass both the token and either an attachment identifier or item identifier to a third-party system."